app/javascript/modules/sidebar/index.js

```diff
@@ -22,19 +22,28 @@ export const handleChangeSidebar = () => {
 
 export const handleChangeNavForm = () => {
   const navFilterForm = elementSelector('nav-filter');
-  navFilterForm.onchange = function () {
-    let sidebarProjectSelectionInput = elementSelector('project-selection');
-    if (sidebarProjectSelectionInput.selectedIndex !== 0){
-      this.submit();
-    } else {
-      const url = window.location.href.split('?')[0];
-      window.location.assign(url);
+  if (navFilterForm != null) {
+    navFilterForm.onchange = function () {
+      let sidebarProjectSelectionInput = elementSelector('project-selection');
+      if (sidebarProjectSelectionInput.selectedIndex !== 0){
+        this.submit();
+      } else {
+        const url = window.location.href.split('?')[0];
+        window.location.assign(url);
+      }
     }
```

Comment:
@santiagovidal @horacio @santib i have one question here: when a navigate the navbar from to select the period is  not present in the next page view so this cause an error because it is trying to subscribe a `onchange` in a null object, is there a better way maybe to bind some js to specifics views or something like that? in order to avoid the `if navFilterForm != null`

Reply:
I'm not a JS guru, but I'm fine with your approach

Reply:
Ditto. LGTM!